### ISSUE TITLE
When Username field is changed to Email address then system allows user to login into the application by entering Username into the email address field.

### ISSUE THREAD

**Issue by Mohtshum** [dnnsoftware/Dnn.Platform#2348]
## Description of bug
When Username field is changed to Email address then system allows user to login into the application by entering Username into the email address field.
For login page, when Username field is changed to Email address then system allows user to login into the application by entering Username into the email address field.

This will happen only one time.

## Steps to reproduce
1.	Login as the superuser when user can login into the application using username. 
2.	Change the settings, to use the email address as username. 
3.	Logout from application. 
4.	Now Enter the username of newly created user/existing user in the email address field. (Refer Precondition) In my case its content editor. 
5.	Enter the password for newly created user.(content editor) 
6.	Hit the login button. 

## Current result
System allows user to enter into the application when we enter Username into the Email address field of login page. 
This happen only once. 
After that if we logged out and try to login in the application again by entering Username into the Email address field then system gives error of Invalid Username/Password. 

## Expected result
System should not allow user to enter into the application when we enter Username into the Email address field of login page.

## Affected version
* [x] 9.2.2
* [x] 9.2.1
* [x] 9.2
* [x] 9.1.1
* [ ] 9.1
* [ ] 9.0

**Comment by valadas** [comment]
I cannot reproduce in 9.4.0 RC or the steps to reproduce are missing something. Please open a new issue if this is still the case in current versions. Thanks.
